libs/components/forms/src/lib/modules/selection-box/selection-box-grid.component.ts

```diff
@@ -49,122 +49,145 @@ export class SkySelectionBoxGridComponent
    * @default 'center'
    */
   @Input()
-  public set alignItems(value: SkySelectionBoxGridAlignItemsType) {
-    this._alignItems = value;
+  public set alignItems(value: SkySelectionBoxGridAlignItemsType | undefined) {
+    if (value) {
+      this.#_alignItems = value;
+    } else {
+      this.#_alignItems = 'center';
+    }
   }
 
   public get alignItems(): SkySelectionBoxGridAlignItemsType {
-    return this._alignItems || 'center';
+    return this.#_alignItems;
   }
 
   @ContentChildren(SkySelectionBoxComponent, {
     read: SkySelectionBoxComponent,
   })
-  public selectionBoxes: QueryList<SkySelectionBoxComponent>;
-
-  private set currentBreakpoint(value: SkyMediaBreakpoints) {
-    if (value !== this._currentBreakpoint) {
-      this._currentBreakpoint = value;
-      this.selectionBoxAdapter.setResponsiveClass(this.elementRef, value);
-      this.updateChildrenHeights();
+  public selectionBoxes: QueryList<SkySelectionBoxComponent> | undefined;
+
+  set #currentBreakpoint(value: SkyMediaBreakpoints) {
+    if (value !== this.#_currentBreakpoint) {
+      this.#_currentBreakpoint = value;
+      this.#selectionBoxAdapter.setResponsiveClass(
+        this.containerElementRef,
+        value
+      );
+      this.#updateChildrenHeights();
     }
   }
 
   @ViewChild('container', {
     read: ElementRef,
     static: true,
   })
-  private elementRef: ElementRef<any>;
+  public containerElementRef!: ElementRef<any>;
+
+  #mutationObserver: MutationObserver | undefined;
 
-  private mutationObserver: MutationObserver;
+  #ngUnsubscribe = new Subject<void>();
 
-  private ngUnsubscribe = new Subject<void>();
+  #_alignItems: SkySelectionBoxGridAlignItemsType = 'center';
 
-  private _alignItems: SkySelectionBoxGridAlignItemsType;
+  #_currentBreakpoint: SkyMediaBreakpoints | undefined;
```

Comment:
This property is set to `| undefined` but its setter's definition is `set #currentBreakpoint(value: SkyMediaBreakpoints)`. Is it possible for this property to be set to undefined? Should it have a default value instead?

Reply:
I made a change around this to give a default. I almost prefer the `| undefined` though as we have to call through the setter anyway to set the default classes and just letting the `ngAfterViewInit` handle this with it just being `undefined` prior seems more efficient. Thoughts?

Reply:
Fixed